src/js/modules/articles/actionTypes.js

```diff
@@ -1,3 +1,6 @@
 export const CREATE_ARTICLE = 'core/CREATE_ARTICLE'
 export const SAVE_SELECTED_ARTICLES = 'core/SAVE_SELECTED_ARTICLES'
-export const DELETE_SELECTED_ARTICLES = 'core/DELETE_SELECTED_ARTICLES'
\ No newline at end of file
+export const DELETE_SELECTED_ARTICLES = 'core/DELETE_SELECTED_ARTICLES'
+export const FETCH_ARTICLES_REQUESTED = "forms/FETCH_ARTICLES_REQUESTED";
+export const FETCH_ARTICLES_SUCCEEDED = 'forms/FETCH_ARTICLES_SUCCEEDED';
+export const FETCH_ARTICLES_FAILED = 'forms/FETCH_ARTICLES_FAILED'
```

Comment:
i dont understand the big picture because it's your code, but you may want to rethink having action types from different modules in the same file/module